jobs/config.json

```diff
@@ -2347,8 +2346,7 @@
     "--env-file=jobs/ci-kubernetes-soak-gce-federation-test.env",
     "--soak-test",
     "--up=false",
-    "--down=false",
-    "--tag=v20170223-43ce8f86"
```

Comment:
https://github.com/kubernetes/test-infra/blob/master/jenkins/bootstrap_test.py#L1935-L1937

We want all soak jobs to use old stable images till code freeze is over. If it will not affect release feel free to make a temporary special case in the test.